apps/api/src/domain/models/vocabulary-list/commands/analyze-term-in-vocabulary-list/vocabulary-list-analyzed-in-term.event.ts

```diff
@@ -0,0 +1,6 @@
+import { BaseEvent } from '../../../shared/events/base-event.entity';
+import { ANALYZE_TERM_IN_VOCABULARY_LIST } from './constants';
+
+export class AnalyzeTermInVocabularyList extends BaseEvent {
```

Comment:
Use the generic on base event and also export `type TermInVocabularyListAnalyzedPayload= AnalyzeTermInVocabularyList`.

Reply:
done